### DISCUSSION TITLE
Android 15  - VanillaIceCream

### DISCUSSION THREAD

**Discussion by ghost** [NightscoutFoundation/xDrip#3493] [Bug reports]
Hi,
It is not possible to install xDrip on my Pixel 7a.
I have the window Usafe app blocked.
![image](https://github.com/NightscoutFoundation/xDrip/assets/6518973/c9acab22-7bcd-4423-b015-55bc85b02b3d)
When I click on install anyway I have a message to explain that the app is not installed. The app is done for a older version.

My understanding was that the last version must be compatible.

Thank
Christian

**Comment by jonprogers** [comment]
I think this is related to discussion #3491

**Reply by ghost** [reply]
You right.  I do not have seen this discussion.
Thanks

**Comment by Navid200** [comment]
This should be resolved now.
Please try the latest Nightly or any after that.

https://github.com/NightscoutFoundation/xDrip/releases/tag/2024.05.29
